zingo-testutils/build.rs

```diff
@@ -0,0 +1,12 @@
+use std::{process::Command};
+
+include!("src/paths.rs");
+
+fn main() {
+    let bin_path = get_cargo_manifest_dir();
+    let _output = Command::new("git")
+        .current_dir(bin_path)
+        .args(["clone", "https://github.com/zingolabs/test_binaries.git"])
```

Comment:
Wait, why is the `clone` in zingo-testutils...   shouldn't it be in build_utils?

Reply:
nah, build-utils is a prereq for zingolib. test-utils is where we want the bin